src/domain/journey/space/pages/AdminSpaceCommunityPage.tsx

```diff
@@ -72,6 +76,15 @@ const AdminSpaceCommunityPage: FC<SettingsPageProps> = ({ routePrefix = '../' })
 
   const currentMembersIds = useMemo(() => users.map(user => user.id), [users]);
 
+  const [dialogOpen, setDialogOpen] = useState(false);
```

Comment:
dialogOpen sounds a bit like an action, I know we have a lot of such usages but isDialogOpen is better.